packages/ui/src/select/index.tsx

```diff
@@ -1,16 +1,21 @@
-'use client';
+"use client";
 
-import styles from './select.css';
-import clsx from 'clsx';
+import { useBottomSheet } from "@swifty/hooks";
+import clsx from "clsx";
+import { useRef } from "react";
+
+import BottomSheet from "../bottom-sheet";
+import Heading from "../heading";
+import styles from "./select.css";
 
 interface SelectProps {
   label?: string;
   placeholder: string;
   options: SelectOption[];
   className?: string;
-  // eslint-disable-next-line no-unused-vars
-  onChange?: 
-    (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  optionLabel: string;
+  value: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
위에서 말씀 드린 SelectType을 만들어두면
```suggestion
className?: string;
options: SelectType[];
```
이렇게 줄일 수 있지 않을까요?